public/docs/_examples/cb-item-template/ts/app/simple-select.component.ts

```diff
@@ -0,0 +1,85 @@
+// #docplaster
+// #docregion
+import { Component } from '@angular/core';
+import { ContentChild } from '@angular/core';
+import { EventEmitter } from '@angular/core';
+import { Input } from '@angular/core';
+import { Output } from '@angular/core';
+import { TemplateRef } from '@angular/core';
```

Comment:
You can add them alphabetically in one line. Or if one line gets too long, you can do one import but putting each item on a new line.